src/octoprint/static/js/app/viewmodels/access.js

```diff
@@ -89,6 +95,7 @@ $(function () {
                     return (
                         self.editor.name() &&
                         self.editor.name().trim() &&
+                        !self.editor.invalidUsername() &&
```

Comment:
Instead of double negation:

```suggestion
                        self.editor.validUsername() &&
```